style.css

```diff
@@ -0,0 +1,97 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+body{
+    font-family: consolas;
+    min-height: 100vh;
+    background-color: #333;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.container{
+    width: 1000px;
+    position: relative;
```

Comment:
Why position relative in container @murtazshaikh

Reply:
to place its child top left according to container.